src/Umbraco.Cms.Integrations.OAuthProxy/Configuration/ServiceConfiguration.cs

```diff
@@ -46,6 +59,10 @@ public static void AddServiceClients(this IServiceCollection services)
             {
                 c.BaseAddress = new Uri("https://login.microsoftonline.com/common/");
             });
+            services.AddHttpClient("AprimoToken", c =>
+            {
+                c.BaseAddress = new Uri($"https://{httpContextAccessor.HttpContext.Request.Headers["tenant"].First()}.aprimo.com/");
```

Comment:
I haven't followed why we need to provide this.  If the customer is using the proxy, they'll be going through our "app" that's registered with Aprimo.  And if that's the case, wouldn't the "tenant" be a single value that never changes?  And if so could be hardcoded in this URL rather that provided in a header?